File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response_map200.dart
part of swagger.api;

class InlineResponseMap200 {
  /* English display name of the locale. */
  String name = null;
/* Display name of the locale in its own language. */
  String localName = null;

  InlineResponseMap200();

  @override
  String toString() {
    return 'InlineResponseMap200[name=$name, localName=$localName, ]';
  }

  InlineResponseMap200.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    name = json['name'];
    localName = json['local_name'];
  }

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'local_name': localName
     };
  }

  static List<InlineResponseMap200> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponseMap200>() : json.map((value) => new InlineResponseMap200.fromJson(value)).toList();
  }

  static Map<String, InlineResponseMap200> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponseMap200>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponseMap200.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/InlineResponseMap200.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponseMap200 {
    /// <summary>
    /// English display name of the locale.
    /// </summary>
    /// <value>English display name of the locale.</value>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// Display name of the locale in its own language.
    /// </summary>
    /// <value>Display name of the locale in its own language.</value>
    [DataMember(Name="local_name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "local_name")]
    public string LocalName { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponseMap200 {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  LocalName: ").Append(LocalName).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
